cypress/support/step-definitions/filter-component-steps.js

```diff
@@ -0,0 +1,5 @@
+import { filterPreview } from '../../locators/filter';
+
+Then('Filter label align property is set to {string}', (align) => {
+  filterPreview().should('have.class', `carbon-filter--align-${align}`);
```

Comment:
carbon-filter--align- - should be as a const in the start of this file

Reply:
fixed